### PR TITLE
🚀 General Availability release

### PR DESCRIPTION
**Pull request by gwyneplaine**
* Replace `@keystone-next` with `@keystone-6` in preparation for namespace change
* Reset versions to `0.0.0` in preparation for the major release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 [Keystone 6](http://keystonejs.com) is the new major version of KeystoneJS, the most powerful headless content management system around. We've substantially rewritten the CLI, Schema config, and Admin UI to make them more powerful and easier to use than ever before.
 
-It's currently in the final stages of preview release, and published on npm under the `@keystone-next` namespace. To learn more, check out our [Roadmap page](https://keystonejs.com/updates/roadmap).
+It's currently in the final stages of preview release, and published on npm under the `@keystone-6` namespace. To learn more, check out our [Roadmap page](https://keystonejs.com/updates/roadmap).
```
Comment (raveling) [inline README.md:33]:
```suggestion
Keystone 6 is published on npm under the `@keystone-6` namespace. To learn about where we’re taking Keystone check out our [Roadmap](https://keystonejs.com/updates/roadmap).
```

**Review by emmatown — APPROVED** [review]

**Review by dcousens — APPROVED** [review]
Go

**Comment by gwyneplaine** [comment]
LGTM

**Review by bladey — APPROVED** [review]

**Review by JedWatson — APPROVED** [review]
Let's Gooooo 🚀

**Review by dcousens — CHANGES_REQUESTED** [review]
Tests failing,  holding until resolved

**Review by dcousens — APPROVED** [review]

**Pull request merged**